Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py

```diff
@@ -0,0 +1,419 @@
+        """Get a list of all your internet exposure assets using the '/assets/get_assets_internet_exposure/' endpoint.
+
+        Args:
+            search_params (list): list of search parameters to add to the API call body.
+
+        Returns:
+            dict: dict containing list of internet exposure assets.
+        """
+        data = {"request_data": {"filters": search_params, "search_to": 100}}
+        headers = self._headers
+
+        response = self._http_request('POST', '/assets/get_assets_internet_exposure/',
+                                      json_data=data, headers=headers)
+
+        return response
+
+    def get_asset_internet_exposure_request(self, asm_id_list: List[str]) -> Dict[str, Any]:
+        """Get internet exposure asset details using the '/assets/get_asset_internet_exposure/' endpoint.
+
+        Args:
+            asm_id_list (list): single attack surface management id in list format.
+
+        Returns:
+            dict: dict containing information on an internet exposure asset.
+        """
+        data = {"request_data": {"asm_id_list": asm_id_list}}
+        headers = self._headers
+
+        response = self._http_request('POST', '/assets/get_asset_internet_exposure/',
+                                      json_data=data, headers=headers)
+
+        return response
+
+
+def get_external_services_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    asm-get-external-services command: Returns list of external services.
+
+    Args:
+        client (Client): CortexAttackSurfaceManagment client to use.
+        args (dict): all command arguments, usually passed from ``demisto.args()``.
+            ``args['ip_address']`` IP Address to search on.
+            ``args['domain']`` Domain to search on.
+            ``args['is_active']`` If the service active or not.
+            ``args['discovery_type']`` how service was discovered.
+
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains external services.
+    """
+    ip_address = args.get('ip_address')
+    domain = args.get('domain')
+    is_active = args.get('is_active')
+    discovery_type = args.get('discovery_type')
+    # create list of search parameters or pass empty list.
+    search_params = []
+    if ip_address:
+        search_params.append({"field": "ip_address", "operator": "eq", "value": ip_address})
+    if domain:
+        search_params.append({"field": "domain", "operator": "contains", "value": domain})
+    if is_active:
+        search_params.append({"field": "is_active", "operator": "in", "value": [is_active]})
+    if discovery_type:
+        search_params.append({"field": "discovery_type", "operator": "in", "value": [discovery_type]})
+
+    response = client.get_external_services_request(search_params)
+    parsed = response.get('reply', {}).get('external_services')
+    markdown = tableToMarkdown('External Services', parsed, removeNull=True)
```

Comment:
Please apply it to all the commands (it will improve the table headers). 
```suggestion
    markdown = tableToMarkdown('External Services', parsed, removeNull=True, headerTransform=string_to_table_header)
```

Reply:
Addressed in Commit `Bulk 10/20/22`